packages/components/datepicker/README.mdx

```diff
@@ -1,27 +1,19 @@
 ---
 title: 'Datepicker'
-status: 'beta'
+status: 'alpha'
 isNew: true
```

Comment:
even though `isNew: true` is added I don't see the `<Badge>New</Badge>` in the sidebar navigation. It should be displayed the same way like it is for Pagination right?

Reply:
This was because the prop wasn't being passed to items on subnavigations on the sidebar, it is fixed now

Reply:
Try again, Vercel didn't build the last commit.